README.md

```diff
@@ -80,7 +80,7 @@ public class Main {
 }
 ```
 
-### API
+### RawKVClient API
```

Comment:
maybe we should delete the api document here:
- 3.1 and 3.2 is different, users may confuse.
- we should deploy the java API document instead of list in README.

Reply:
> maybe we should delete the api document here:
> 
> * 3.1 and 3.2 is different, users may confuse.
> * we should deploy the java API document instead of list in README.

Agree.